cdi/pom.xml

```diff
@@ -225,6 +234,35 @@
                     <skip>false</skip>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.moditect</groupId>
+                <artifactId>moditect-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>add-module-infos</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>add-module-info</goal>
+                        </goals>
+                        <configuration>
+                            <module>
+                                <moduleInfo>
+                                    <name>${java.module.name}</name>
+                                    <exports>
+                                        !org.hibernate.validator.*.internal;
+                                        !org.hibernate.validator.*.internal.*;
+                                        *;
+                                    </exports>
```

Comment:
Can't you have this part in common (parent POM) while overriding only `<opens>` in modules where it makes sense?